.github/workflows/build.yml

```diff
@@ -2,7 +2,8 @@ name: Deploy
 
 on:
   push:
-    branches: [ $default-branch ]
+    branches:
+      - master
```

Comment:
@ojeytonwilliams This should be changed to `main`.

Reply:
Agreed.  I'll do that and update the action.